src/nvim/ui.h

```diff
@@ -13,14 +13,22 @@ typedef enum {
   kUIPopupmenu,
   kUITabline,
   kUIWildmenu,
+  kUIWidgetCount,
 } UIWidget;
```

Comment:
could rename this to `UIExtension`

Reply:
I would prefer the former, there is not necessarily any control involved in all extensions.